kitsune/search/v2/views.py

```diff
@@ -1,17 +1,28 @@
 import json
 from math import ceil
+from typing import List
 
 from django.conf import settings
 from django.http import HttpResponse
-from django.utils.translation import ugettext as _, pgettext
+from django.utils.translation import pgettext
+from django.utils.translation import ugettext as _
 
 from kitsune import search as constants
-from kitsune.search.forms import SimpleSearchForm
-from kitsune.sumo.api_utils import JSONRenderer
 from kitsune.products.models import Product
-from kitsune.search.views import _fallback_results
+from kitsune.search.forms import SimpleSearchForm
 from kitsune.search.utils import locale_or_default
 from kitsune.search.v2.search import CompoundSearch, QuestionSearch, WikiSearch
+from kitsune.search.views import _fallback_results
+from kitsune.sumo.api_utils import JSONRenderer
+
+
+def _get_product_title(product_title: str) -> List[str]:
```

Comment:
Shouldn't this annotation be `-> tuple[Product, list[str]]`?

Reply:
good point. I added that before switching to tuple. We need to set mypy in pre-commit hook for starters :)